stock_secondary_unit/views/stock_picking_views.xml

```diff
@@ -23,27 +23,11 @@
                                 '&amp;', ('product_tmpl_id.product_variant_ids', 'in', [product_id]),
                                          ('product_id', '=', False)]"
                     options="{'no_create': True}"
-                    attrs="{'column_invisible': [('parent.immediate_transfer', '=', True)], 'readonly': ['|', ('is_initial_demand_editable', '=', False), '&amp;', '&amp;', ('show_operations', '=', True), ('is_locked', '=', True), ('is_initial_demand_editable', '=', False)]}"
+                    groups="uom.group_uom"
+                    readonly="not is_initial_demand_editable"
```

Comment:
Same here

Reply:
@pilarvargas-tecnativa Hi! You are correct about the readonly! I just corrected it. Regarding column_invisible, the immediate_transfer field no longer exists in version 17 of the stock.picking model.